site/static/layouts/tucked-menu-vertical/index.html

```diff
@@ -5,12 +5,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="A set of horizontal menus that switch to vertical and which hide at small window widths.">
     <title>Responsive Horizontal-to-Vertical Menu &ndash; Layout Examples &ndash; Pure</title>
-    <link rel="stylesheet" href="https://unpkg.com/purecss@1.0.1/build/pure-min.css">
-    <link rel="stylesheet" href="https://unpkg.com/purecss@1.0.1/build/grids-responsive-min.css">
+    <link rel="stylesheet" href="/css/pure/pure-min.css">
+    <link rel="stylesheet" href="/css/pure/grids-responsive-min.css">
 </head>
 <body>
 
 <style>
+html {
+    font-family: sans-serif;
+}
```

Comment:
It looks like already done in `base.css` https://github.com/pure-css/pure/pull/790/files#diff-2ddcd75fbddbbf98ed56fdd9c23071d2

Anything I'm missing?

Reply:
You are right. I missed cleaning this one up. Will remove it.